### DISCUSSION TITLE
.PY file in ./templates dir bug ? or no ?

### DISCUSSION THREAD

**Discussion by Namba-ir** [miguelgrinberg/microdot#268] [Q&A]
In my templates folder, when I run my software, a series of .py files are created with the same name as the files of my website pages. Is this normal?! Or does it indicate that a part of my software has a problem or has a bug or error that creates these files?

- ![Screenshot 2024-11-18 200724](https://github.com/user-attachments/assets/30f64c0f-0f85-476e-8b60-3acec7937371)

**Comment by miguelgrinberg** [comment]
This is normal, these are the compiled code for your uTemplate templates.

**Comment by Namba-ir** [comment]
thank you very much
